ocs_ci/deployment/aws.py

```diff
@@ -61,6 +61,20 @@ def __init__(self):
         # check or during a cluster cleanup)
         self.cluster_prefixes_special_rules = CLUSTER_PREFIXES_SPECIAL_RULES
 
+    def deploy_ocp(self, log_cli_level="DEBUG"):
+        super(AWSBase, self).deploy_ocp(log_cli_level)
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+
+        if ocs_version >= version.VERSION_4_10 and ocp_version >= version.VERSION_4_9:
+            if config.DEPLOYMENT.get("gp2_csi"):
+                self.DEFAULT_STORAGECLASS = "gp2-csi"
+            elif config.DEPLOYMENT.get("gp2"):
+                self.DEFAULT_STORAGECLASS = "gp2"
+            # The default storage class is gp3-csi for OCS 4.10 and above
+            else:
+                self.DEFAULT_STORAGECLASS = "gp3-csi"
```

Comment:
if we go with my above suggestions, the conditions here would look like:
```python
            if config.DEPLOYMENT.get("customized_deployment_storage_class"):
                self.DEFAULT_STORAGECLASS = config.DEPLOYMENT.get("customized_deployment_storage_class")

```

Reply:
Will do.